### PR TITLE
Add a template for the local volume provisioner

### PR DESCRIPTION
**Pull request by nfnt**
JIRA: [DCOS-55514](https://jira.mesosphere.com/browse/DCOS-55514)

### CODE REVIEW

**Comment by shaneutt** [comment]
@nfnt we need signed commits, can you squash and sign please?

**Comment by hectorj2f** [comment]
We're testing this approach, let's wait before merging it.

**Comment by nfnt** [comment]
Okay, I'm squasing and signing then.

**Comment by alejandroEsc** [comment]
I apologize if this is known, but where is the chart from? I was reading it and i have some comments on it.

**Comment by nfnt** [comment]
@alejandroEsc The chart is adapted from the [helm templates provided by the storage local static provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/helm). Could you add your comments on the PR for the chart https://github.com/mesosphere/charts/pull/2?

**Comment by hectorj2f** [comment]
@nfnt The helm chart should be now available using the helm repo/version/chart...

**Comment by shaneutt** [comment]
@nfnt & @gpaul LGTM to looks like we might have a few more considerations as per recent updates to mesosphere/charts.

File: templates/localvolumeprovisioner.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: localvolumeprovisioner
+  namespace: kubeaddons
+spec:
+  kubernetes:
+    minSupportedVersion: v1.14.0
+  chartReference:
+    chart: https://mesosphere.github.io/charts/stable/localvolumeprovisioner-0.1.0.tgz
+    # FIXME: Use the repo instead, once the index issues are resolved.
+    #chart: localvolumeprovisioner
```
Comment (hectorj2f) [inline templates/localvolumeprovisioner.yaml]: @gpaul We could revert this commented code now. It should work

Comment (nfnt) [reply templates/localvolumeprovisioner.yaml]: Done!

**Pull request merged**